File: programmers/lv_1/K번째수.py
def solution(array, commands):
    answer = []
    
    for command in commands:
        i = command[0]
        j = command[1]
        k = command[2]
        
        splitted_arr = array[i-1:j]
        splitted_arr.sort()
        answer.append(splitted_arr[k-1])
    
    return answer

File: programmers/lv_1/K번째수.java
import java.util.*;

class Solution {
    public int[] solution(int[] array, int[][] commands) {
        int comLen = commands.length;
        int[] answer = new int[comLen];
        
        int i, j, k;
        int iter = 0;
        for (int[] command : commands) {
            i = command[0];
            j = command[1];
            k = command[2];
            
            int[] newArr = Arrays.copyOfRange(array, i - 1, j);
            Arrays.sort(newArr);
            answer[iter] = newArr[k - 1];
            
            iter++;
        }
        
        return answer;
    }
}